addons/base_import/models/base_import.py

```diff
@@ -712,12 +712,33 @@ def _remove_currency_symbol(self, value):
             # Otherwise it is not a float with a currency symbol
             return False
 
+    def _convert_scientific_notation_to_float(self, value):
+        sci_pattern = re.compile('-?\ *[0-9]+\ *[.,]?\ *[0-9]*\ *(?:[Ee]\ *[0-9]*\ *[+-]?\ *[0-9]+)')
+        sci_list = [x for x in re.findall(sci_pattern, value)]
+
+        # if no pattern => not a sci notation
+        if not len(sci_list):
+            return value
+
+        # if too many patterns => probably a string or not a proper sci notation
+        if len(sci_list) > 1:
+            return False
+
+        value = value.replace(' ', '').replace(',', '.')
```

Comment:
This only takes into account the French version in which the space is used as thousands separator, and the comma is used as decimal separator. But in others versions you can use a dot or an apostrophe as thousands separator. In these cases I'm not sure this will work. It's for these reasons that a function to infer the separators exists :  _infer_separators.

See: https://en.wikipedia.org/wiki/Decimal_separator#Examples_of_use